### DISCUSSION TITLE
Image regression using current version

### DISCUSSION THREAD

**Discussion by hasansari** [Project-MONAI/MONAI#1401] [Q&A]
Is it possible to do image regression (for example MR to CT conversion using MAE as loss function) using the current version of MONAI? Are there any examples/tutorials available?

**Comment by Nic-Ma** [comment]
Hi @wyli ,

I think we have some tracking ticket and plan for `image regression`, could you please help share more info?

Thanks in advance.

**Reply by wyli** [reply]
thanks I created a ticket https://github.com/Project-MONAI/tutorials/issues/100

**Reply by royak** [reply]
Is there any tutorial by now?

**Reply by KumoLiu** [reply]
Hi @royak, here is a demo for MRI reconstruction, it has a regression-like pipeline.
https://github.com/Project-MONAI/tutorials/tree/main/reconstruction/MRI_reconstruction
Thanks!

**Reply by royak** [reply]
Thank you!
